synapseclient/client.py

```diff
@@ -790,14 +865,15 @@ def getUserProfile(
             )
         )
 
-    def _findPrincipals(self, query_string: str) -> typing.List[UserGroupHeader]:
+    def _findPrincipals(self, query_string: str) -> List[UserGroupHeader]:
         """
         Find users or groups by name or email.
 
-        :returns: A list of userGroupHeader objects with fields displayName, email, firstName, lastName, isIndividual,
-                  ownerId
+        Returns:
+            A list of userGroupHeader objects with fields displayName, email, firstName, lastName, isIndividual, ownerId
+
 
-        Example::
+        Example: Find userGroupHeader objects for test user
 
             syn._findPrincipals('test')
```

Comment:
Can you double check how this is going to render on the mkdocs site. You might need to tab in once more so it looks something like:


```
Example: Title for this example
        
        code here
```

You may also have to do something like:

```
Example: Title for this example
    Additional context
        
        code here
```

The formatting requirement is a bit more strict on this piece

Reply:
Gotcha. It might be worthwhile making a testing ground page while going through this just to verify formatting. If we add this to the docs site we will want the formatting correct.

Reply:
I updated the format of the Examples and tested it for one public method: `synapseclient.table.PartialRowset` The formatting in the docs site is changed 

From 
<img width="779" alt="Screenshot 2024-01-08 at 11 20 02 PM" src="https://github.com/Sage-Bionetworks/synapsePythonClient/assets/90745557/7b09285a-171d-45ce-9d25-2d1d8fbad024">

to 

<img width="958" alt="Screenshot 2024-01-08 at 11 48 47 PM" src="https://github.com/Sage-Bionetworks/synapsePythonClient/assets/90745557/b65dc383-9e33-4dca-a81e-56a7081b8c6e">

Reply:
Awesome! Thanks for checking this